lib/msal-browser/src/interaction_client/PopupClient.ts

```diff
@@ -97,10 +97,17 @@ export class PopupClient extends StandardInteractionClient {
             const authClient: AuthorizationCodeClient = await this.createAuthCodeClient(serverTelemetryManager, validRequest.authority, validRequest.azureCloudOptions);
             this.logger.verbose("Auth code client created");
 
+            const isNativeBroker = NativeMessageHandler.isNativeAvailable(this.config, this.logger, this.nativeMessageHandler, request.authenticationScheme);
+            // Start measurement for server calls with native brokering enabled
+            let fetchNativeAccIdMeasurement;
```

Comment:
Can we spell out "Account" here?

Reply:
Sure. Was trying to save length but it is long anyway. :)